hazelcast/src/main/java/com/hazelcast/config/alto/AltoSocketConfig.java

```diff
@@ -4,55 +4,48 @@
 
 import javax.annotation.Nonnull;
 import java.util.Objects;
-import java.util.regex.Pattern;
 
 public class AltoSocketConfig {
-    private static final String DEFAULT_PORT_RANGE = "11000-21000";
-    private static final int DEFAULT_RECEIVE_BUFFER_SIZE = 128 * 1024;
-    private static final int DEFAULT_SEND_BUFFER_SIZE = 128 * 1024;
-
-    private String portRange = DEFAULT_PORT_RANGE;
-    private int receiveBufferSize = DEFAULT_RECEIVE_BUFFER_SIZE;
-    private int sendBufferSize = DEFAULT_SEND_BUFFER_SIZE;
+    private String portRange = "11000-21000";
+    private int receiveBufferSizeKb = 128;
+    private int sendBufferSizeKb = 128;
 
     public String getPortRange() {
         return portRange;
     }
 
     public AltoSocketConfig setPortRange(@Nonnull String portRange) {
-        if (!Bounds.PORT_RANGE_PATTERN.matcher(portRange).matches()) {
-            throw new InvalidConfigurationException("Invalid port definition");
+        if (!portRange.matches("\\d{1,5}-\\d{1,5}")) {
```

Comment:
what about portrange 

100-90.

Reply:
It will throw HZ exception with "Allowed alto socket port range isn't enough.". Added a test for that.

https://github.com/hazelcast/hazelcast/pull/23319/commits/605f389a4fd5ff432fc5251f3e1c76a546d9c9da

Reply:
Javadoc is missing with respect to bad port range string.

Reply:
And when is this exception thrown? When the config is made or when the Hazelcast instance starts with the broken config? Best to validate as part of the config creation.

Reply:
I do not see any added documentation.

Reply:
There is documentation. It's either Github messing up or you reviewed the commit here and not the PR. See `AltoSocketConfig` from files section in this PR.